core/modules/views/templates/views.theme.inc

```diff
@@ -680,45 +679,27 @@ function template_preprocess_views_view_table(&$variables) {
  */
 function template_preprocess_views_view_grid(&$variables) {
   $view     = $variables['view'];
+  $rows     = $variables['rows'];
   $result   = $view->result;
-  $options  = $view->style_plugin->options;
-  $handler  = $view->style_plugin;
+  $style    = $view->style_plugin;
+  $options  = $style->options;
```

Comment:
I know this is in order to make this more pretty/readable, but I think that it is against our coding standards.